src/tests/ViewModel_test.py

```diff
@@ -4,31 +4,29 @@
 from todolist.Task import Task, Status
 
 dateformat = "%Y-%m-%dT%H:%M:%S.%fZ"
+dummyNotToday = datetime.strptime("2020-02-18T08:15:27.243860Z", dateformat)
+dummyToday = datetime.strptime("2020-06-14T09:17:20.141163Z", dateformat)
 
 ### TODO: rewrite these to not fail for the first 2 hours of the day
-def test_correct_recent_done_items():
+def test_correct_recent_done_items(monkeypatch):
     # Given
-    task1DoneAt = datetime.now() - timedelta(minutes=15)
-    task2DoneAt = datetime.now() - timedelta(minutes=10)
-    task3DoneAt = datetime.strptime("2020-01-08T08:15:27.243860Z", dateformat)
-    task4DoneAt = datetime.strptime("2020-06-29T09:17:20.141163Z", dateformat)
-    task5DoneAt = datetime.now() - timedelta(minutes=1)
-    task6DoneAt = datetime.now() - timedelta(minutes=120)
+    monkeypatch.setattr(Task, "touchedOn", lambda self, date: self.last_modified == dummyToday)
```

Comment:
Your change seems a bit confusing/overcomplicated.

You could have just used the real function but set today/notToday equal to `datetime.now()` and `datetime.now() - timedelta(days=1)`

If your reasoning was that unit tests should mock other classes' functions, this is going about it a bit wrong. You aren't checking that touchedOn is used correctly (the date param is totally ignored in your monkeypatch function) and by using dummyToday/dummyNotToday, it's not immediately obvious to what extent the date matters in the test. I would keep it as `touched_today`, and just mock it to return true/false based on the id or something. 
But I don't think mocking is necessary I don't think is necessary for a little function like Task.touchedToday. Task is still basically just a data class.

I would say in general you should just use the real datetime and if there are problems, you should mock the current time (freezegun does it for Python). If you wanted to test a variety of times like before this PR, then you could just get the real date and set the time to 1 am, 11pm etc.